tests/.pug-lintrc

```diff
@@ -0,0 +1,19 @@
+{
+"disallowAttributeInterpolation": true,
+"disallowClassAttributeWithStaticValue": true,
+"disallowDuplicateAttributes": true,
+"disallowHtmlText": true,
+"disallowIdAttributeWithStaticValue": true,
+"disallowLegacyMixinCall": true,
+"disallowMultipleLineBreaks": true,
+"disallowSpacesInsideAttributeBrackets": true,
+"requireClassLiteralsBeforeAttributes": true,
+"requireIdLiteralsBeforeAttributes": true,
+"requireLowerCaseAttributes": true,
+"requireLowerCaseTags": true,
+"requireSpaceAfterCodeOperator": true,
+"requireStrictEqualityOperators": true,
+"validateAttributeSeparator": { "separator": ", ", "multiLineSeparator": ",\n    " },
+"validateDivTags": true,
+"validateTemplateString": true
+}
```

Comment:
I'll add the key. We have so many files at the root level now that I was trying to avoid adding another 🍂